pkg/build_deb.sh

```diff
@@ -81,13 +81,14 @@ fi
 ##
 
 mkdir -p $ROOT/DEBIAN
-echo "9" >$ROOT/DEBIAN/compat
+#echo "9" >$ROOT/DEBIAN/compat
```

Comment:
`debhelper` is not used, see [COMPATIBILITY LEVELS](http://manpages.debian.org/cgi-bin/man.cgi?query=debhelper&apropos=0&sektion=0&manpath=Debian+8+jessie&format=html&locale=en).